packages/eslint-plugin-pf-codemods/lib/rules/v5/overflowMenuDropdownItem-renamed-prop.js

```diff
@@ -0,0 +1,41 @@
+const { getPackageImports } = require("../../helpers");
+
+// https://github.com/patternfly/patternfly-react/pull/8359
+module.exports = {
+  meta: { fixable: "code" },
+  create: function (context) {
```

Comment:
Not a blocker, but I'm curious why you didn't just use the rename props helper?

Reply:
may have been because of the custom message - but one of the helpers allows for that as well.